stockflux-securities-master/src/components/securities-table/TableRow.js

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import classNames from "classnames";
+import ToolTip from "../tool-tip/ToolTip";
+import {
+  FaPen,
+  FaTrashAlt,
+  FaEye,
+  FaEyeSlash,
+  FaCheck,
+  FaTimes
+} from "react-icons/fa";
+import PropTypes from "prop-types";
+import "./TableRow.css";
+
+const TableRow = ({ item, deleteSecurity, patchSecurity }) => {
+  const handleDelete = () => {
+    deleteSecurity(item.securityId);
+  };
+
+  const handleToggleVisible = () => {
+    patchSecurity(item.securityId, {
+      visible: !item.visible
+    });
+  };
+
+  const handleToggleEnabled = () => {
+    patchSecurity(item.securityId, {
+      enabled: !item.enabled
+    });
+  };
```

Comment:
`handleToggleVisible` and `handleToggleVisible` are slightly similar. Maybe something like this would help reduce repetition:
```
  const handleToggle = (feature) => {
    patchSecurity(item.securityId, {
      [feature]: !item[feature]
    });
  };
```
Not sure if this would work though, needs testing. And again, being very picky here.

Reply:
This can work but it requires the button's `onClick` to be called inline like `onClick={() => handleToggle("enabled")}`

I think @joelwilliamsmith advised against using a button's `onClick` in this way.